docs/pages/material-ui/api/accordion-actions.json

```diff
@@ -13,6 +13,8 @@
   "name": "AccordionActions",
   "styles": { "classes": ["root", "spacing"], "globalClasses": {}, "name": "MuiAccordionActions" },
   "spread": true,
+  "themeDefaultProps": true,
+  "muiName": "MuiAccordionActions",
```

Comment:
I think this should not happen with MUI Base right? @oliviertassinari

Reply:
This does not seem right. https://deploy-preview-36490--material-ui.netlify.app/base/api/badge-unstyled/#theme-default-props

Reply:
Right, oops

Reply:
Fixed, it should be clearer now.